src/contracts/atlas/Escrow.sol

```diff
@@ -303,7 +305,8 @@ abstract contract Escrow is AtlETH {
 
         // Verify that we can lend the solver their tx value
         if (solverOp.value > address(this).balance) {
-            return (1 << uint256(SolverOutcome.CallValueTooHigh), gasLimit);
+            result |= 1 << uint256(SolverOutcome.CallValueTooHigh);
```

Comment:
Line 300-302 could use the function `solverGasLimitScaledDown()`:
https://github.com/FastLane-Labs/atlas/blob/spearbit-reaudit-wk-2/src/contracts/libraries/AccountingMath.sol#L29-L40